src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Details.cshtml

```diff
@@ -167,7 +167,7 @@
                     <th scope="row" class="govuk-table__header das-table-cell-width-45">Employer</th>
                     <td id="apprenticeship-employer" class="govuk-table__cell das-table-cell-width-40">@Model.Employer</td>
                     <td class="govuk-table__cell das-table-cell-width-15 govuk-table__cell--numeric">
-                        @if (Model.ShowChangeEmployerLink)
+                        @if (Model.ShowChangeEmployerLink && !Model.IsOnFlexiPaymentPilot.GetValueOrDefault())
```

Comment:
Not sure whether we have a preference on whether we'd do this or actually embed the condition within "ShowChangeEmployerLink" so that we can unit test it?

Reply:
Ye I did consider this but it's also used to determine whether or not to show the "help with change of circumstances" partial so I left it.